lib/tasks/db.js

```diff
@@ -297,6 +297,8 @@ function getSequelizeInstance () {
 function getMigrator (type, callback) {
   if (helpers.config.configFileExists() || args.url) {
     var sequelize = getSequelizeInstance();
+    var coffeeRegex = /^(?!skeleton)(.*)\.js$|\.coffee$/;
```

Comment:
Maybe something like [this](http://rubular.com/r/Z0b2npfbQf):

```
^(?!skeleton).+(.*)\.(js|coffee)$
```

It excludes any files starting with skeleton while still matching js and coffee files.